### ISSUE TITLE
Learning rate multiplier on mapping layers (StyleVectorizer)

### ISSUE THREAD

**Issue by bob80333** [lucidrains/stylegan2-pytorch#84]
In the official tensorflow implementation, the learning rate for the mapping layers is multiplied by 0.01.

See here:
https://github.com/NVlabs/stylegan2/blob/master/training/networks_stylegan2.py#L260

In order to allow this, the FC layers are reimplemented manually here:
https://github.com/NVlabs/stylegan2/blob/master/training/networks_stylegan2.py#L41

I couldn't find anything similar in this repo, although I might have missed it.

**Comment by lucidrains** [comment]
@bob80333 yup, I noticed that, but it hasn't made a big difference for me in terms of results

**Comment by bob80333** [comment]
Ah, it doesn't make much of a difference? Interesting.  Well, in that case, I'll close this unless I find some scenario where it does make a difference.

**Comment by lucidrains** [comment]
@bob80333 every paper comes with a few big main ideas, and then a bunch of tiny tweaks
